guilford_court_commitment.yaml

```diff
@@ -0,0 +1,408 @@
+            value: ""
+      ol.description: # most serious offense code
+        type: "ol.description"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "MOST SERIOUS OFFENSE CODE"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "MOST SERIOUS OFFENSE CODE"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+    name: "charges"
+
+  supervisions:
+    fqn: "ol.supervision"
+    entitySetName: "GuilfordSupervisions"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["P\  CASE INTAKE DATE", "OFFENDER NC DOC ID NUMBER"]
+      ol.recognizedstartdate: # p&p commitment status flag
+        type: "ol.recognizedstartdate"
+        column: "P&P CASE INTAKE DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+      ol.status: # p&p commitment status flag
+        type: "ol.status"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "P&P COMMITMENT STATUS FLAG"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "P&P COMMITMENT STATUS FLAG"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      general.datetime: # p&p commitment status date
+        type: "general.datetime"
+        column: "P&P COMMITMENT STATUS DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+      ol.durationdays: # total length of supervision
+        type: "ol.durationdays"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "TOTAL LENGTH OF SUPERVISION"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "TOTAL LENGTH OF SUPERVISION"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      ol.type:
+        type: "ol.type"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "TYPE OF OLD PE DATE CODE"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "TYPE OF OLD PE DATE CODE"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      ol.newperiod:
+        type: "ol.newperiod"
+        column: "NEW PERIOD OF SUPERVISION FLAG"
+    name: "supervisions"
+
+  locations:
+    fqn: "ol.location"
+    entitySetName: "GuilfordLocations"
+    propertyDefinitions:
+      general.id: # most serious offense code
+        type: "general.id"
+        column: "CO OF CONV MOST SERIOUS OFFNSE"
```

Comment:
Is this a typo or the orginal column name?

Reply:
this is the original column name.